client/src/lang/translations/nl.json

```diff
@@ -0,0 +1,93 @@
+  "modal-delete-file-header": "Verwijder bestand?",
+  "modal-delete-file-text": "Weet je zeker dat je {0} wilt verwijderen?",
+  "modal-preview-file-header": "Voorbeeld",
+  "modal-preview-file-click-to-zoom": "Klik om te zoomen",
+  "abort": "Annuleer",
+  "delete": "Verwijder",
+  "modal-change-password-header": "Wijzig wachtwoord",
+  "old-password": "Oud wachtwoord",
+  "new-password": "Nieuw wachtwoord",
+  "confirm-new-password": "Bevestig nieuw wachtwoord",
+  "new-password-does-not-match": "Nieuw wachtwoord komt niet overeen",
+  "password-does-not-match": "Wachtwoord komt niet overeen",
+  "save": "Bewaren",
+  "today": "Vandaag",
+  "search-label": "Zoeken",
+  "full-date": "{day}-{month}-{year}",
+  "last-edited": "Laatste wijziging:",
+  "password-change-successful": "Wachtwoord succesvol gewijzigd",
+  "no-valid-file": "Geen geldig bestand! Is dit misschien een map?",
+  "search-string-too-short": "Te kort - vereist minstens 3 tekens",
+  "encryption-error-try-relogin": "Fout bij decoderen! Misschien opnieuw proberen in te loggen?",
+  "change-password": "Wijzig wachtwoord",
+  "export-data": "Exporteer data (gedecodeerd)",
+  "import-data": "Reimporteer data (herstellen)",
+  "login-header": "Login",
+  "username-label": "Gebruikersnaam",
+  "password-label": "Wachtwoord",
+  "confirm-password-label": "Bevestig wachtwoord",
+  "login-button": "Login",
+  "registration-successful-please-login": "Registratie geslaagd. Log in!",
+  "register-header": "Registeer",
+  "register-button": "Registeer",
+  "not-yet-saved": "Wijzigingen nog niet opgeslagen!",
+  "calendar-locale": "nl",
+  "modal-history-title": "Geschiedens van geselecteerde dag",
+  "use-selected-history-as-default": "Stel geselecteerde geschiedenisversie in als nieuwste tekst?",
+  "yes": "Ja",
+  "no-history-available-yet": "Er is nog geen geschiedenis aanwezig!",
+  "export-started-1": "Zip-bestand wordt gemaakt. Dit kan enkele minuten duren, afhankelijk van de grootte.",
+  "export-started-2": "BELANGRIJK: Herlaad de pagina NIET tijdens dit proces!",
+  "close": "Sluiten",
+  "download": "Downloaden",
+  "modal-update-available-header": "Nieuwere versie van DailyTxT beschikbaar!",
+  "update-available-client-version": "Huidig geinstalleerde versie",
+  "update-available-recent-version": "Nieuwste versie",
+  "error-uploading-file": "Probleem bij uploaden! Vermoedelijk is het bestand te groot.",
+  "import-data-header": "Opnieuw importeren van data",
+  "import-data-text-1": "Upload progressie:",
+  "import-data-text-2": "Encryptie en importeren van gegevens",
+  "import-successful": "Importeren gelukt",
+  "backup-codes": "Back-up codes",
+  "create-backup-codes": "Genereer back-up codes",
+  "create-new-backup-codes": "Genereer nieuwe back-up codes",
+  "backup-codes-description-1": "Back-up codes werken als eenmalige wachtwoorden. Ze kunnen altijd worden gebruikt in plaats van het normale wachtwoord (bijvoorbeeld als je het vergeten bent), maar slechts één keer.",
```

Comment:
Back-up codes werken als eenmalige wachtwoorden. Je kunt ze altijd gebruiken in plaats van je gewone wachtwoord (bijvoorbeeld als je het bent vergeten), maar slechts één keer.